docs/moderator-handbook.md

```diff
@@ -1,88 +1,88 @@
+FreeCodeCamp is an inclusive community, and we need to keep it that way.
 
-We have a single code of conduct that governs our entire community. The fewer the rules, the easier they are to remember. You can read those rules and commit them to memory [here](https://code-of-conduct.freecodecamp.org).
+We have a single [Code of Conduct](https://code-of-conduct.freecodecamp.org) that governs our entire community. The fewer the rules, the easier they are to remember. You can read those rules and commit them to memory [here](https://code-of-conduct.freecodecamp.org).
 
 > [!NOTE]
-> As a moderator we would add you to one or more teams on GitHub, our community forum(s) & chat servers. If you are missing access on a platform that you would like to moderate please [reach out to a staff member](FAQ.md#additional-assistance).
+> As a moderator we would add you to one or more teams on GitHub, our community forums & chat servers. If you are missing access on a platform that you would like to moderate, please [reach out to a staff member](FAQ.md#additional-assistance).
 
 ## Moderating GitHub
 
 Moderators have two primary responsibilities on GitHub:
 
-1. Triaging and responding to issues
-2. Reviewing and merging pull requests (a.k.a QA).
+1. Triaging and responding to issues.
+2. Reviewing and merging pull requests (aka QA).
 
-### Moderating GitHub Issues
+### Moderating GitHub issues
 
-We use our main [`freeCodeCamp/freeCodeCamp`](https://github.com/freeCodeCamp/freeCodeCamp/issues) repo as a common issue tracker for all of our repositories. We get new issues every day, all of which need to be triaged, labeled and addressed. This is also a great place to start helping with open-source codebase contributions.
+We use our main [`freeCodeCamp/freeCodeCamp`](https://github.com/freeCodeCamp/freeCodeCamp/issues) repository as a common issue tracker for all of our repositories. We get new issues every day, all of which need to be triaged, labeled, and addressed. This is also a great place to start helping with open-source codebase contributions.
 
-#### Issue Triage
+#### Issue triage
 
 [Triaging](https://en.wikipedia.org/wiki/Triage) is a process of prioritizing attention to each new issue report. We have an extensive list of labels that we use to mark each issue's priority, category, status, and scope.
 
 You can help us organize and triage the issue reports by applying labels from [this list](https://github.com/freeCodeCamp/freeCodeCamp/labels). Usually, a description is available alongside the label explaining its meaning.
 
-Please pay special attention to the labels `"help wanted"` and `"first timers only"`. These are to be added to threads that you think can be opened up to potential contributors for making a pull request.
+Please pay special attention to the labels `"help wanted"` and `"first-timers only"`. These are to be added to threads that you think can be opened up to potential contributors for making a pull request.
```

Comment:
```suggestion
Please pay special attention to the labels `"help wanted"` and `"first timers only"`. These are to be added to threads that you think can be opened up to potential contributors for making a pull request.
```

the label is "first timers only"

Reply:
For this, wouldn't the labels have to be updated to fit to "first-timers only"? I plan on reading through the rest of freeCodeCamp's documentation to fix any inconsistent punctuation with a dictionary (first-timer is a compound noun).

Reply:
Talked with Naomi about any labels regarding the words first-time and first-timer, they will be left unhyphenated.